<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="mer_table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>250</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>商品图片</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>商品名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>单价</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>购买数量</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="total_label">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>250</y>
     <width>201</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>总金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>订单编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="con_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>500</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>收货人信息：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>订单状态：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
